### PR TITLE
Fixes #18640 - realign templates structure

### PR DESCRIPTION
**Pull request by ares**
Based on #4320 to avoid rebasing problems later. What this does
- move files in app/views/unattended according to structure in community templates
- updates `script/sync_templates.sh` to work with the new structure of community templates
- update all hardcoded paths and names accordingly
- syncs up with current community-templates version

We can't get rid of hardcoded paths just yet. Next step will be syncing metadata from Foreman to community-templates so we have os_family for partition tables and template kinds pro provisioning templates set correctly. Then we can just iterate over files and do the import based on what we find in metadata.

I verified that seed and file are in sync by comparing SEEDED_TEMPLATES and files I found using Dir.glob. Also I tried doing the seed on empty DB, all seems to fit. The same applies to partition tables.

### CODE REVIEW

File: app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    # static network configuration
+    ip = @host.provision_interface.ip
+    mask = subnet.mask
+    gw = subnet.gateway
+    dns = [subnet.dns_primary]
+    if subnet.dns_secondary != ''
+      dns.push(subnet.dns_secondary)
+    end
+    if (@host.operatingsystem.name.match(/Fedora/i) && major < 17) || major < 7
+      # old Anacoda found in Fedora 16 or RHEL 6 and older
```
Comment (dLobatog) [inline app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb:39]: nitpick 'Anaconda'

**Comment by dLobatog** [comment]
I didn't find any problems in terms of the number of templates moved. I didn't check the content but I trust it'll be fine - or you can send PRs afterwards to fix it. Check out tests in foreman & katello though @ares 😄

File: app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb (hunk 1 above)
Comment (ares) [reply app/views/unattended/provisioning_templates/PXEGrub/kickstart_default_pxegrub.erb:39]: Must be fixed in community-templates -> https://github.com/theforeman/community-templates/pull/376

**Comment by ares** [comment]
Foreman tests failures are unrelated HostsControllerTest.test_0070_select multiple action with valid host_ids param should return a selection page on pg/ruby 2.0 and 2.3

**Comment by dLobatog** [comment]
Thanks @ares !

**Pull request merged**